include/grpc/support/slice_buffer.h

```diff
@@ -40,6 +40,8 @@
 extern "C" {
 #endif
 
+#define GRPC_SLICE_BUFFER_INLINE_ELEMENTS 8
+
```

Comment:
Looks good. It will be fun to experiment with this constant to see the crossover between saved mallocs and increased memcpys.